styles/prosilver/template/js/scoring_preview.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
While fixing this, I found a bug in my code.  My JS code should not be executed/included unless the WOLFSBLVT_POLL_SCORING var is on.  So, we have two options now: leave the code in a separate file, as is, and wrap it around <-- IF -->, or merge the file with yours and change coding to detect if the corresponding DOM element exists before executing my code.
I prefer the former (clearer and cleaner), but the latter is ok as well, and would reduce the number of files.  What do you prefer?